.github/workflows/windows-msvc.yml

```diff
@@ -182,6 +182,7 @@ jobs:
           sed -i '/AT_SETUP(\[EXTERNAL RELATIVE file])/a AT_SKIP_IF(\[true\])' testsuite.src/run_file.at
           sed -i '/AT_SETUP(\[LINE SEQUENTIAL one Record])/a AT_SKIP_IF(\[true\])' testsuite.src/run_file.at
           sed -i '/AT_SETUP(\[Concatenated Files])/a AT_SKIP_IF(\[true\])' testsuite.src/run_file.at
+          sed -i '/AT_SETUP(\[FUNCTION RANDOM])/a AT_SKIP_IF(\[true\])' testsuite.src/run_functions.at
```

Comment:
does that function fail in MPIR or what is the issue?

Reply:
Freezes randomly (maybe for the same reason as in the GC3 CI).